resources/guis.yml

```diff
@@ -256,7 +256,7 @@ guis:
   - Linux
   price: Free
   license: GNU GPL
-  order: 28
+  order: 4
```

Comment:
Likewise, I wonder if we are picking up non-Git uses of "Guitar" to produce such a huge jump.